website/static/js/osfHelpers.js

```diff
@@ -658,6 +658,14 @@ var _confirmationString = function() {
 };
 
 /**
+*  Helper function to judge if the user browser is IE
+*/
+var isIE = function(userAgent) {
+  userAgent = userAgent || navigator.userAgent;
```

Comment:
> Use 4 spaces for indentation

https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#style

Will fix this when I merge.